lib/components/ClinicUserSelect.js

```diff
@@ -41,6 +42,10 @@ var ClinicUserSelect = React.createClass({
     }
   },
 
+  handleOnChange: function(userId) {
+    this.props.setTargetUser(userId, {eventName: metrics.CLINIC_SEARCH_SELECTED});
+  },
```

Comment:
So I guess I see the issue (raised in an earlier comment) about the lack of tests for this. The root cause of the problem is how we don't yet have unit tests around React components in the uploader (only around the redux implementation). The tests will be pretty easy to add once we're set up for them, but setting up for React tests _and_ writing them may significantly expand the scope of this task.

Curious as to your thoughts on this @jh-bate, balancing "scope creep" against realizing that if we merge as-is we're not including tests for the addition of some metrics which we've tried a bit harder in the uploader to do (thus far).